### PR TITLE
Project first task

### PR DESCRIPTION
**Pull request by tmaxe**
Oleksiy Prosyanko

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (oleksiypr) [inline .idea/.gitignore]: Please check gitignore for python projects  example: https://gist.github.com/GhostofGoes/94580e76cd251972b15b4821c8a06f59

File: .idea/ctj_edu.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oleksiypr) [inline .idea/ctj_edu.iml]: This is IDE related file, should not been committed (subject of gitignore)

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (oleksiypr) [inline .idea/inspectionProfiles/profiles_settings.xml]: This is IDE related file, should not been committed (subject of gitignore)

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oleksiypr) [inline .idea/misc.xml]: This is IDE related file, should not been committed (subject of gitignore)

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oleksiypr) [inline .idea/modules.xml]: This is IDE related file, should not been committed (subject of gitignore)

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oleksiypr) [inline .idea/vcs.xml]: This is IDE related file, should not been committed (subject of gitignore)

**Review by oleksiypr — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (tmaxe) [reply .idea/.gitignore]: Is it OK now?

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
```
Comment (oleksiypr) [inline .gitignore]: Please add blank line to the end of the file

File: cgi-bin/hello.py [hunk 8]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+print("Content-type: text/html")
+print()
+print("<h1>Hello Python!</h1>")
```
Comment (oleksiypr) [inline cgi-bin/hello.py]: Please add blank line to the end of the file

File: server.py [hunk 9]
```diff
@@ -0,0 +1,4 @@
+from http.server import HTTPServer, CGIHTTPRequestHandler
+server_address = ("", 8000)
+httpd = HTTPServer(server_address, CGIHTTPRequestHandler)
+httpd.serve_forever()
```
Comment (oleksiypr) [inline server.py:4]: Please add blank line to the end of the file

**Review by oleksiypr — CHANGES_REQUESTED** [review]

File: cgi-bin/hello.py (hunk 8 above)
Comment (tmaxe) [reply cgi-bin/hello.py]: Added

File: .gitignore (hunk 7 above)
Comment (tmaxe) [reply .gitignore]: Added

**Review by oleksiypr — APPROVED** [review]

**Pull request merged**